observability/logging/logging-6.1/log6x-loki.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[leveloffset=+1]
+[id="log6x-loki"]
+= Storing logs with LokiStack
```

Comment:
```suggestion
= Configuring LokiStack storage
```

Reply:
This will be taken up in https://issues.redhat.com/browse/OBSDOCS-1494